.github/workflows/tag.yml

```
@@ -53,4 +53,14 @@ jobs:
         uses: azure/CLI@v1.0.6
         with:
           inlineScript: |
-            az storage blob upload-batch -s dist -d \$web --account-name ${{ steps.variables.outputs.AZURE_ACCOUNT_PROD }}
\ No newline at end of file
+            az storage blob upload-batch -s dist -d \$web --account-name ${{ steps.variables.outputs.AZURE_ACCOUNT_PROD }}
+
+      - name: Create Release
+        # Only create a release on one brand to avoid duplicate error
+        if: contains(matrix.brand, 'swedbankpay')
+        uses: softprops/action-gh-release@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          name: Design Guide v${{ steps.variables.outputs.VERSION }}
+          body_path: RELEASE_NOTES.md
```

Isn't the file being changed for every release called `CHANGELOG.md`?

```suggestion
          body_path: CHANGELOG.md
```

---

Ah right. I was thinking about this without writing a comment about it, my bad.
My thoughts were: For this PR, what if we introduce a `RELEASE_NOTES.md` file that only contains the latest changes, while the `CHANGELOG.md` contains everything. This can then be automated in the future.

---

Yeah, that makes sense. 👍🏼